roles/usegalaxy-eu.bashrc/files/galaxy_jwd.py

```diff
@@ -75,20 +91,29 @@ def main():
         raise ValueError("Please set ENV PGHOST")
 
     # Check if ~/.pgpass file exists and is not empty
-    if not os.path.isfile(os.path.expanduser("~/.pgpass")) or os.stat(os.path.expanduser("~/.pgpass")).st_size == 0:
-        raise ValueError("Please create a ~/.pgpass file in format: <pg_host>:5432:*:<pg_user>:<pg_password>")
+    if (
+        not os.path.isfile(os.path.expanduser("~/.pgpass"))
+        or os.stat(os.path.expanduser("~/.pgpass")).st_size == 0
+    ):
+        raise ValueError(
+            "Please create a ~/.pgpass file in format: <pg_host>:5432:*:<pg_user>:<pg_password>"
+        )
 
     # Check if the given galaxy.yml file exists
     if not os.path.isfile(os.environ.get("GALAXY_CONFIG_FILE")):
-        raise ValueError(f"The given galaxy.yml file {os.environ.get('GALAXY_CONFIG_FILE')} does not exist")
+        raise ValueError(
+            f"The given galaxy.yml file {os.environ.get('GALAXY_CONFIG_FILE')} does not exist"
```

Comment:
wouldn't it be better to define a variable once and than access it, instead of always pull out the ENV from os.environ?

Reply:
1st block checks if the ENVs exist and the code below collects all of them in their respective variables. I can also merge both blocks into one so this repetition does not happen.

Reply:
Fixed in new commit https://github.com/usegalaxy-eu/infrastructure-playbook/pull/662/commits/beee7aa6f49d1465d29c877e5c92a6352edd0c83